apps/demo/story-config/CustomSelectField.config.tsx

```diff
@@ -0,0 +1,136 @@
+import {DemoConfiguration} from "@config";
+import {CustomSelectFieldDemo} from "@stories";
+import {Box, CustomSelectField} from "ferns-ui";
+import React from "react";
+
+export const CustomSelectFieldConfiguration: DemoConfiguration = {
+  name: "Custom Select Field",
+  component: CustomSelectField,
+  related: ["SelectField", "Field", "Tap-to-edit"],
+  description:
+    "Displays a list of options using the browser’s native select and includes a custom option that renders a text field when selected and allows user to input value not included in predefined options.",
+  a11yNotes: ["The list should be labeled so that screen readers know that the list is related."],
+  category: "Component",
+  status: {
+    // TODO: ensure custom select component meets accessibility requirements
```

Comment:
Custom select in particular is complex in terms of accessibility. I made my best first pass at improvements, and I think we should get this merged in so it doesn't block migration and come back to it.